moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -203,20 +206,16 @@ void JogCalcs::run(const ros::TimerEvent& timer_event)
   tf_moveit_to_cmd_frame_ = kinematic_state_->getGlobalLinkTransform(parameters_.planning_frame).inverse() *
                             kinematic_state_->getGlobalLinkTransform(parameters_.robot_link_command_frame);
 
-  // Input frame determined by YAML file if not passed with message
-  if (twist_stamped_.header.frame_id.empty())
-  {
-    twist_stamped_.header.frame_id = parameters_.robot_link_command_frame;
-  }
-
   have_nonzero_command_ = have_nonzero_twist_stamped_ || have_nonzero_joint_jog_;
 
+  // Don't end this function without updating the filters
+  updated_filters_ = false;
+
   // If paused or while waiting for initial jog commands, just keep the low-pass filters up to date with current
   // joints so a jump doesn't occur when restarting
   if (wait_for_jog_commands_ || paused_)
   {
-    for (std::size_t i = 0; i < num_joints_; ++i)
-      position_filters_[i].reset(original_joint_state_.position[i]);
+    resetLowPassFilters(original_joint_state_);
```

Comment:
I think you could streamline the logic here. Lines 218, 240, 248, 263, 317 all make the same call to update the filters. Can we depend on L317 to handle it?

Reply:
A rebase since your comment changed the line numbers, but I have removed one of the mentioned calls to the filters. The one shown above and 2 of the other ones come directly before early returns from the `run()` function, so I think they will need to stay